evaporate.js

```diff
@@ -208,9 +218,22 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
            cancelAllRequests();
         };
 
+        me.pause = function (){
+          l.d('Paused FileUpload ', me.id);
+          setStatus(PAUSED);
+          cancelAllRequests();
```

Comment:
`cancelAllRequests` stops active uploads, as well as aborting the multipart upload. I would consider "pausing" to entail stopping active uploads but retaining any already uploaded multipart file parts in S3. Maybe you could use a different function here instead of `cancelAllRequests` that stops active uploads without aborting the multipart upload:

```
            function cancelActiveUploads() {
                l.d('cancelActiveUploads()');

                for (var i = 1; i < parts.length; i++) {
                    abortPart(i, true);
                }
            }
```